tests/Unit/PointwiseFunctions/EquationsOfState/Test_Tabulated.cpp

```diff
@@ -0,0 +1,135 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <string>
+#include <vector>
+
+#include "DataStructures/Matrix.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"  // IWYU pragma: keep
+#include "IO/H5/ReadTabulated.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "tests/Unit/TestingFramework.hpp"
+
+namespace {
+
+void test_tabulated() noexcept {
+  h5::H5File<h5::AccessType::ReadOnly> sample_file(unit_test_path() +
+                                                   "/IO/KDE_Sample.h5");
```

Comment:
I don't know what `KDE_Sample` means (it's also not camel case). I think a more useful name would be `StellarCollapse2010Sample.h5`

Reply:
Stellar collapse has a number of different tabulated equations of state. I thought it would be good to reference exactly which one of the 12 or so different ones we are getting the data from for the sample file. The prefix for the one I chose is KDE. I could explain that in a comment.